src/pages/cities/Forms/CityForm.js

```diff
@@ -7,173 +7,157 @@ import toast from 'react-hot-toast'
 import { RequestsContext } from 'src/providers/RequestsContext'
 import { useInvalidate } from 'src/hooks/resource'
 import { ResourceIds } from 'src/resources/ResourceIds'
+import { useForm } from 'src/hooks/form'
 
 // ** Custom Imports
 import CustomTextField from 'src/components/Inputs/CustomTextField'
 
 import { SystemRepository } from 'src/repositories/SystemRepository'
 import ResourceComboBox from 'src/components/Shared/ResourceComboBox'
 
-export default function CityForm ({ labels, recordId, maxAccess }) {
+export default function CityForm({ labels, recordId, maxAccess }) {
+  const [isLoading, setIsLoading] = useState(false)
+  const [editMode, setEditMode] = useState(!!recordId)
 
-    const [isLoading, setIsLoading] = useState(false)
-    const [editMode, setEditMode] = useState(!!recordId)
+  const [initialValues, setInitialData] = useState({
```

Comment:
can't you remove this state and add the initial values directly to the formik?

Reply:
yes